axonserver/src/main/java/io/axoniq/axonserver/message/command/CommandCache.java

```diff
@@ -105,6 +105,15 @@ public CommandInformation put(@Nonnull String key, @Nonnull CommandInformation v
         return map.put(key, value);
     }
 
+    /**
```

Comment:
Do you still want to allow the normal put() operation here?

Reply:
Same applies for the QueryCache

Reply:
Removed in the new commit: `put` is deprecated in the child-interface and an exception throwing default implementation is provided.